### PR TITLE
1st commit

### PR DESCRIPTION
**Pull request by rahulkanade04**
Code changed

### CODE REVIEW

File: src/gitdemo/abcd.java [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
-
+int a=10;
```
Comment (bharatbhosle) [inline src/gitdemo/abcd.java:7]: add int b = 20;

**Review by bharatbhosle — COMMENTED** [review]
Hi Rahul,
Please add one more int value in abcd class

**Comment by rahulkanade04** [comment]
2 nd commit

**Review by bharatbhosle — APPROVED** [review]
Hi Rahul,
Changes done are ok.
approved
pls go ahead with merge.

Regards,
Bharat

**Pull request merged**